policies/guest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Name": "azure-operator",
+  "Description": "Role for github.com/giantswarm/azure-operator",
+  "Actions": [
+    "*"
+  ],
+  "NotActions": [
+    "Microsoft.Authorization/elevateAccess/Action"
+  ],
+  "AssignableScopes": [
+    "/subscriptions/${SUBSCRIPTION_ID}"
```

Comment:
Planning to use `os.ExpandEnv` to expand those variables later on.
But this also make this file not usable by azure cli as their is no way to pass the variable. Some shell magic would be needed there.

Reply:
I think that's fine.

Reply:
I do not understand how this is used but in general I think this should be a go template then. Fucking with the environment is only a pain in the ass most of the time and later we have to refactor it anyway.

Reply:
I like the go template idea. This can be regular file still. Not necessary a `.go` file. You can name it `guest.tmpl.json` or something.

Reply:
When it is effectively a template then it should be `guest.json.tmpl`, but yeah.

Reply:
`guest.json.tmpl` is impractical, because of editors highlighting.

Reply:
Your editor should be configurable and detect that properly. The thing is that it is semantically more correct to express the file type with its extension. Go templates aren't valid JSON files by nature. So this would raise quite wrong expectations. You have to render the template in order to get a valid JSON as it should be.

Reply:
Let's say I can configure my editor which is unecessary.
But I can't configure github.
And this is a common practice. Check for example files in this directory: https://github.com/giantswarm/azure-operator/tree/master/helm/azure-operator-chart/templates
We do that pretty much in every single project.